contracts/payloads/IGP81/PayloadIGP81.sol

```diff
@@ -0,0 +1,503 @@
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 25_000_000 * 1e18 // 25M shares // $50M
+            });
+
+            IFluidDex(sUSDe_USDT_DEX_ADDRESS).updateUserSupplyConfigs(config_);
+        }
+
+        {
+            // [TYPE 2] sUSDe-USDT<>USDT | smart collateral & debt
+            Vault memory VAULT_sUSDe_USDT_USDT = Vault({
+                vault: sUSDe_USDT__USDT_VAULT,
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 22_500_000, // $22.5M
+                maxBorrowLimitInUSD: 45_000_000 // $45M
+            });
+
+            setVaultLimits(VAULT_sUSDe_USDT_USDT); // TYPE_2 => 92
+        }
+
+    }
+
+    // @notice Action 3: Set dust allowance for cbBTC-USDT DEX T4 vault
+    function action3() internal {
+        address cbBTC_USDT_DEX_ADDRESS = getDexAddress(22);
+        address cbBTC_USDT__cbBTC_USDT_VAULT_ADRESS = getVaultAddress(105);
+
+        {
+            // dust limits
+            Dex memory DEX_cbBTC_USDT = Dex({
+                dex: cbBTC_USDT_DEX_ADDRESS,
+                tokenA: cbBTC_ADDRESS,
+                tokenB: USDT_ADDRESS,
+                smartCollateral: true,
+                smartDebt: true,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 20_000 // $20k
+            });
+            setDexLimits(DEX_cbBTC_USDT); // Smart Collateral & Smart Debt
+
+            DEX_FACTORY.setDexAuth(cbBTC_USDT_DEX_ADDRESS, TEAM_MULTISIG, true);
+        }
+        
+        {// Set team multisig as vault auth for cbBTC_USDT T4 Vault
+            VAULT_FACTORY.setVaultAuth(
+                cbBTC_USDT__cbBTC_USDT_VAULT_ADRESS,
+                TEAM_MULTISIG,
+                true
+            );
+        }
+
+    }
+
+    // @notice Action 4:  Reduce max supply shares and max borrow limit for USDe-USDT<>USDT
+    function action4() internal {
+        address USDe_USDT_DEX_ADDRESS = getDexAddress(18);
+        address USDe_USDT__USDT_VAULT = getVaultAddress(93);
+        
+        {
+            // Dexcrease Max Supply Shares
+            IFluidDex(USDe_USDT_DEX_ADDRESS).updateMaxSupplyShares(
+                10_000_000 * 1e18 // 10M shares // $20M
+            );
+        }
+
+        {
+            // Update USDe-USDT<>USDT vault supply shares limit
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: USDe_USDT__USDT_VAULT,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 10_000_000 * 1e18 // 10M shares // $20M
```

Comment:
we should reduce to 7.5M$ or 10M$ IMO. which is 3.75M shares / 5M shares

Reply:
<img width="1115" alt="Screenshot 2025-01-31 at 3 57 19 @KaymasJain src="https://github.com/user-attachments/assets/169239a5-6329-42b0-81f5-298cc6f4af9b" />
@KaymasJain can you pls confirm here

Reply:
so base limit for withdraw should not be max supply caps. that has the effect of essentially having no withdraw limit as all the way up to the cap would be fully withdrawable always 

so while cap can be 10M shares, base limit should be significantly lower. benefit of a high base limit is better UX as the vault fills up

Reply:
noted!